features/02_contact.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Send a Message through the contact button
+
+ As a common user
+    I want to use the contact button
+    In order to send a message
+
+Scenario: Sending messages through the contact button
+        Given the user is on contact button
```

Comment:
this should be on the contact form (but is not the best practice to modify this given)
this should be the same given you created for the login.